package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "web3-onboard-monorepo",
-  "version": "2.5.0",
+  "version": "2.5.1",
```

```suggestion
  "version": "2.5.0",
```
We will handle this version bump on production release

---

We will need the version bumped in the injected package.json [here](https://github.com/blocknative/web3-onboard/blob/e71f10fe45946d2a4e2fb5fb98db00991b64ab72/packages/injected/package.json#L3) and you will only need to bump the alpha version i.e. version will become `2.0.15-alpha.2`